usr/src/lib/libsmbfs/smb/acl_api.c

```diff
@@ -83,6 +85,7 @@ smbfs_acl_iocget(int fd, uint32_t selector, mbdata_t *mbp)
 		return (error);
 
 	m = mbp->mb_top;
+	bzero(&iocb, sizeof (iocb));
```

Comment:
Elsewhere in this library you use memset() instead of bzero.  I think memset is preferable in userland, but probalby the code should at least be self consistent if we're going to use bstrings.

Reply:
Probably just too lazy to type
(void) memset(p, 0, sizeof(*p))